packages/ui/src/components/Alert/alert.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Alert as DefaultAlert, AlertProps as DefaultAlertProps } from 'antd'
+import classNames from 'classnames'
+
+type CondoAlertProps = {
+    type: 'success' | 'info' | 'warning' | 'error';
+}
+
+const ALERT_CLASS_PREFIX = 'condo-alert'
+
+export type AlertProps = Omit<DefaultAlertProps, 'type' | 'prefix' | 'prefixCls' | 'banner' | 'afterClose' | 'closable' | 'closeText' | 'closeIcon' | 'onClose'> & CondoAlertProps
+
+const Alert: React.FC<AlertProps> = (props) => {
+    const { type, className, ...rest } = props
+
+    const classes = classNames(
+        {
+            [`${ALERT_CLASS_PREFIX}-${type}`]: type,
```

Comment:
Antd have this classes by default, so no need to override it here
https://github.com/ant-design/ant-design/blob/4.x-stable/components/alert/index.tsx#L153